solidity/random-beacon/test/fixtures/index.ts

```diff
@@ -1,18 +1,21 @@
 import { Contract } from "ethers"
-import { ethers } from "hardhat"
+import { ethers, getNamedAccounts } from "hardhat"
 import type {
   SortitionPool,
   SortitionPoolStub,
   RandomBeaconStub,
   RandomBeaconGovernance,
   StakingStub,
 } from "../../typechain"
+import { to1e18 } from "../functions"
 
 export const constants = {
   groupSize: 64,
   groupThreshold: 33,
   signatureThreshold: 48, // groupThreshold + (groupSize - groupThreshold) / 2
   offchainDkgTime: 72, // 5 * (1 + 5) + 2 * (1 + 10) + 20
+  minimumStake: to1e18(100000),
+  poolWeightDivisor: 2000,
```

Comment:
I think it's `to1e18(1)`, please confirm with the version currently deployed on mainnet for ECDSA stakers.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2707/commits/c97dba585f021ce3e8ec7fb37e2665a4f7577541 as part of #2707